action.yml

```diff
@@ -44,13 +55,13 @@ runs:
       #   3. package.json#packageManager
       #   4. default pnpm version (^8.0.0)
       run: |
-        if [ ! -f "package.json" ]; then 
+        if [ ! -f "${{inputs.node-version-file}}" ]; then
           echo "Could not find package.json. Did actions/checkout run?"
           exit 1
         fi
 
         forcedVersion=${{inputs.pnpm-version}}
-        packageJson=$(cat package.json)
```

Comment:
this will error if node-version-file is not a json file.

node-version-file is meant for setup-node, rather than just the package.json. it can be an .nvmrc, .node-version, etc

Reply:
maybe instead, the working-directory should prefix?